components/MovieCard/IMovieData.tsx

```diff
@@ -0,0 +1,57 @@
+interface IProdCompany {
```

Comment:
- For interface declaration you do not need JSX. So change the extension from `tsx` to `ts`
- These are types not data. Call the file `MovieCard.types.ts`